lib/jekyll/convertible.rb

```diff
@@ -76,34 +78,23 @@ def validate_permalink!(filename)
     #
     # Returns the transformed contents.
     def transform
-      converters.reduce(content) do |output, converter|
-        begin
-          converter.convert output
-        rescue => e
-          Jekyll.logger.error(
-            "Conversion error:",
-            "#{converter.class} encountered an error while converting '#{path}':"
-          )
-          Jekyll.logger.error("", e.to_s)
-          raise e
-        end
-      end
+      _renderer.transform
     end
 
     # Determine the extension depending on content_type.
     #
     # Returns the String extension for the output file.
     #   e.g. ".html" for an HTML output file.
     def output_ext
-      Jekyll::Renderer.new(site, self).output_ext
+      _renderer.output_ext
     end
 
     # Determine which converter to use based on this convertible's
     # extension.
     #
     # Returns the Converter instance.
     def converters
-      @converters ||= site.converters.select { |c| c.matches(ext) }.sort
```

Comment:
Hey! It's been ages since you commented, but I'm circling back now. A new PR (or an issue) will definitely get more of a response than a comment on a really old PR. But here we are!

So you're saying we took away `extname` and everything exploded?